src/mongo_logger.ts

```diff
@@ -254,6 +256,18 @@ export interface LogConvertible extends Record<string, any> {
   toLog(): Record<string, any>;
 }
 
+/** @internal */
+export function maybeTruncate(
```

Comment:
Can we move EJSON stringify calls into the same function that worries about enforcing the size limit? maybeTruncate -> stringifyWithLength (or whatever name seems cooler)